cdap-docs/reference-manual/source/http-restful-api/http-restful-api-v3/namespace.rst

```diff
@@ -46,15 +46,15 @@ Properties for the namespace are passed in the JSON request body:
    * - Parameter
      - Description
      - Default Value (if not defined)
-   * - ``name``
-     - Display name for the namespace
-     - The Namespace ID
    * - ``description``
      - Display description of the namespace
      - An empty string ("")
+   * - ``config``
+     - :ref:`Configuration preferences <preferences>` for the namespace
```

Comment:
@awholegunch I don't think there is a relation between Preferences/RuntimeArguments and these configuration parameters. We should remove this link.